cmd/setup/29.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE IF EXISTS projections.idp_templates6_oauth2 ADD COLUMN IF NOT EXISTS use_pkce BOOLEAN;
```

Comment:
I'd put that into the same file as the oidc table

Reply:
Thanks, I will